zmsbackend/src/Zmsbackend/Process/Service/Process.php

```
@@ -523,6 +523,13 @@ protected function addSearchConditions(\BO\Zmsbackend\Process\Repository\Process
         if (isset($parameter['amendment']) && $parameter['amendment']) {
             $query->addConditionAmendment($parameter['amendment']);
         }
+        if(array_key_exists('customTextfield', $parameter) && $parameter['customTextfield'] !== null && trim((string)) $parameter['customTextfield'] !== '') {
```

PHPStan (2.2.6)

[error] Syntax error, unexpected ')' on line 526


(phpstan.parse)